brainscore_language/plugins/sg_tse/test.py

```diff
@@ -0,0 +1,53 @@
+from pytest import approx
+from brainscore_language import load_benchmark, load_metric
+from brainscore_language.models.huggingface import HuggingfaceSubject
+from brainscore_language.plugins.sg_tse import _load_suite
+from brainscore_language.plugins.sg_tse.data import user_def_params
+
+class TestData:
+    def test_firstline(self):
+        suite = _load_suite(user_def_params.syntaxgymtestsuite())
+        data = list(suite.iter_sentences())
+        assert data[0] == 'The dog sleeps on the mat'
+
+    def test_length(self):
+        suite = _load_suite(user_def_params.syntaxgymtestsuite())
+        data = list(suite.iter_sentences())
+        assert len(data[0]) == 25
+
+class TestMetric:
+    def test_all_correct(self):
+        predictions = ["True", "True", "True"]
+        targets = ["True", "True", "True"]
+        metric = load_metric(user_def_params.metric_id())
```

Comment:
begone `user_def_params`

Reply:
user_def_params 
is now but a memory
in naught but haiku